CHANGELOG.rst

```diff
@@ -14,6 +14,19 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[0.4.0] - 2016-11-23
```

Comment:
@haikuginger beat you to this one 😄 -- it will have to be v0.5.0 or higher by the time this branch merges